### PR TITLE
Testing front

### PR DESCRIPTION
**Pull request by KiltedDrummer**
tests for all components, no end to end tests in this suite

### CODE REVIEW

File: client/src/containers/reservations/FindTableContainer.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ const mapDispatchToProps = (dispatch) => ({
 });
 
 const mapStateToProps = (state) => {
-  name: state.nameFromState
 };
```
[the remark this answers is not in the document]
Comment (KiltedDrummer) [reply client/src/containers/reservations/FindTableContainer.js]: indeed. I'm using it as a place holder for now, I'll either be filling it in when I figure out what i will exactly need, or removing it if it's never used.

File: test/server.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+// afterAll(() => {
+//   browser.close();
+// });
+
+describe('this is a test', () => {
+  test('a dumb test', () => {
+    
+  });
+});
```
Comment (brandonjunus) [inline test/server.test.js]: Any reason for this file at the moment too? Is it just a place holder?

File: testing/dbGenTests.test.js [hunk 3]
```diff
@@ -1,116 +0,0 @@
-const generator = require('../db/dbGeneration');
```
Comment (brandonjunus) [inline testing/dbGenTests.test.js]: Just wondering what happened to this file. It looks like it was placed into a new file with the same name? Was there an issue with this test?

File: test/server.test.js (hunk 2 above)
Comment (KiltedDrummer) [reply test/server.test.js]: I don't have any end-to-end tests yet, this is just a test to allow the file to pass. Will be filled in after I finish the calendar

File: testing/dbGenTests.test.js (hunk 3 above)
Comment (KiltedDrummer) [reply testing/dbGenTests.test.js]: i'm not sure I get the question. 
This line is importing all the dbGeneration functions from dbGeneration.js into the test file

Comment (brandonjunus) [reply testing/dbGenTests.test.js]: Oh it looks like according to the git file changes, the git this entire file was deleted and replaced with a similar file. Maybe its just some strange git issue. My comment is not related to line 1, but the entire dbGenTest.test.js in general.

**Pull request merged**